springfox-spring-web/src/main/java/springfox/documentation/spring/web/readers/parameter/ParameterRequiredReader.java

```diff
@@ -42,10 +43,12 @@
 import java.util.Set;
 import java.util.function.Predicate;
 
+import javax.validation.constraints.NotNull;
+
 import static java.util.Optional.*;
 
 @Component
-@Order(Ordered.HIGHEST_PRECEDENCE)
+@Order(Ordered.LOWEST_PRECEDENCE)
```

Comment:
In order for this plugin to control whether the parameter is required, it must be applied last, not first.

Reply:
This is not desirable. The spring web has to be the highest precedence since it should be applied first.